frontend/e2e/$lang+/_public/apply/adult/_adult-flow.spec.ts

```diff
@@ -0,0 +1,234 @@
+import type { Page } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+
+// Reusable funtion to check empty input
+async function hasError(page: Page, error: string) {
+  await page.getByRole('button', { name: 'Continue' }).click();
```

Comment:
I would remove this line since the submit action name could be different per screens, of course it's not the case right now but who know.